### PR TITLE
Remove broken links in project/index.md

### LINKED ISSUE #9496: [Docs] 404s for "Community" and "Contributing" links on /project
### Current State

In the screenshot below, both `Community` and `Contributing` links are 404s.

### Desired State

Are they even supposed to be links? If so, then point to the correct URLs. If not, remove the links.

I believe the `Community` link should point to `https://docs.meshery.io/project/community`, not `https://docs.meshery.io/community`. This would however be a duplicate of the link `Meshery Community` just below.

As for `Contributing`, it seems like `https://docs.meshery.io/project/contributing` is the right link. However, that page is pretty much the same information as what we already see in the list below `Contributing`.

In the end, removing both links seems like the best solution.

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by dmarcoux**
**Notes for Reviewers**

This PR fixes #9496

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by dmarcoux** [comment]
I don't know if signed commits refer to signed with a GPG key, if so, then yes, the commit is signed. The [link to the explanation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin) points to an empty contributing guide.

**Comment by l5io** [comment]
🚀 Preview for commit f4da6f4e1ba5468e9046413e443918233be3c691 at: https://656796a2dee9ea0973e31fe4--meshery-docs-preview.netlify.app

File: docs/pages/project/index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ list: exclude
 ---
 
 <div class="wrapper">
-  <a href="community"><div class="overview">Community</div></a>
+  <div class="overview">Community</div>
```
Comment (leecalcote) [inline docs/pages/project/index.md:14]: "project/community" is what we seem to be missing.

File: docs/pages/project/index.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ list: exclude
 </ul>
 
 <div class="wrapper"> 
-  <a href="contributing"><div class="overview">Contributing</div></a>
+  <div class="overview">Contributing</div>
```
Comment (leecalcote) [inline docs/pages/project/index.md:29]: "/project/contributing"

**Review by leecalcote — COMMENTED** [review]
Good stuff here, @dmarcoux

File: docs/pages/project/index.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/project/index.md:14]:
```suggestion
  <a href="/project/community"><div class="overview">Community</div></a>
```

File: docs/pages/project/index.md (hunk 2 above)
Comment (leecalcote) [reply docs/pages/project/index.md:29]:
```suggestion
  <a href="/project/contributing"><div class="overview">Contributing</div></a>
```

**Review by leecalcote — APPROVED** [review]
@dmarcoux, this is super helpful. 🙌  💯

**Comment by leecalcote** [comment]
Whoo-hoo, @dmarcoux! 🎈 

The first of many, we can only hope. 🤞

**Pull request merged**